lib/octopus/proxy.rb

```diff
@@ -243,7 +256,7 @@ def clean_connection_proxy
 
     def check_schema_migrations(shard)
       OctopusModel.using(shard).connection.table_exists?(
-        ActiveRecord::Migrator.schema_migrations_table_name,
+          ActiveRecord::Migrator.schema_migrations_table_name,
```

Comment:
I tried to remove it but could not figure out what is causing this.

______________

Mihir Gore
+91-9980618819

*You are what your deep, driving desire is. As your desire is, so is your
will. As your will is, so is your deed. As your deed is, so is your
destiny. --Brihadaranyaka Upanishad IV.4.5*
_______________

On Thu, Aug 13, 2015 at 12:41 PM, Akshay Gupta <notifications@github.com>
wrote:

> In lib/octopus/proxy.rb
> <https://github.com/interviewstreet/octopus/pull/1#discussion_r36946201>:
>
> > @@ -243,7 +256,7 @@ def clean_connection_proxy
> >
> >      def check_schema_migrations(shard)
> >        OctopusModel.using(shard).connection.table_exists?(
> > -        ActiveRecord::Migrator.schema_migrations_table_name,
> > +          ActiveRecord::Migrator.schema_migrations_table_name,
>
> why the extra indent?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/interviewstreet/octopus/pull/1/files#r36946201>.
>